spark/src/main/scala/geotrellis/spark/io/slippy/FileSlippyTileReader.scala

```diff
@@ -16,16 +16,27 @@
 
 package geotrellis.spark.io.slippy
 
+import geotrellis.vector._
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.tiling.SpatialKey
 import geotrellis.spark._
+import geotrellis.spark.io.hadoop._
+import geotrellis.spark.io.hadoop.formats._
 import geotrellis.util.Filesystem
 
 import org.apache.commons.io.FileUtils
 import org.apache.commons.io.filefilter._
 import org.apache.spark._
 import org.apache.spark.rdd._
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.mapreduce.Job
+import org.apache.hadoop.mapreduce.{InputSplit, TaskAttemptContext}
+import org.apache.hadoop.fs.Path
 
 import java.io._
-import scala.collection.JavaConverters._
+import scala.collection.JavaConversions._
```

Comment:
It is deprecated, can you revert your changes back here and use `scala.collection.JavaConverters._`